device-types/Datacom/DM4610-OLT-4GPON+4GX+2XS.yaml

```diff
@@ -0,0 +1,47 @@
+---
+manufacturer: Datacom
+model: DM4610 OLT 4GPON+4GX+2XS
+slug: dm4610-olt-4gpon+4gx+2xs
+part_number: 800.5285.xx
+u_height: 1.0
+is_full_depth: true
+subdevice_role: parent
```

Comment:
`subdevice_role: parent` is meant for devices that contain other devices. Such as a chassis device like the Dell VRTX that then has other devices like the Dell M320 that slots into the VRTX. This `subdevice_role: parent` is not meant for a device that has modules. So, is this actuall a chassis device? If not, please remove that definition entierly.

Reply:
Hi,
Okay, Alright, I'll remove it.
Equipment has bay only for PSU and FAN.
https://www.datacom.com.br/pt/produtos/gpon/dm4610-4gpon

Reply:
Yup those are modules then, not device bays so no need for subdevice_role